transforms/remove-create-reducer.js

```diff
@@ -0,0 +1,59 @@
+export default function transformer( file, api ) {
```

Comment:
This should account for persistence. See https://github.com/Automattic/wp-calypso/blob/898d753d472349bf611509ce128068b38937040e/client/state/utils/create-reducer.js#L29

Reply:
Done.